### PR TITLE
Workbench documents - Sections 12 & 13 of issue 862

### PR DESCRIPTION
**Pull request by JamieDawson**
Here are the workbench documents from https://github.com/nimbella-corp/nimbella-cli/pull/327
Per @joshuaauerbachwatson, request, I have created a new pull request in this repo.

### CODE REVIEW

**Comment by joshuaauerbachwatson** [comment]
Since we are in phase of just getting some missing documents drafted, I'm not going to pick a lot of nits.  All of our documents will likely be wordsmithed and revised going forward.

But, I'd like to see two things corrected before I merge this:

1.  There are broken image links in `workbench-playground-1-intro.md`.   The images should be in the repo and the links should be relative. 
2.  The document `workbench-playground-3-deploy-from-github.md` is misnamed.   It is about a workbench feature but not a playground feature.  

I also note (@JamieDawson, you don't have to fix this prior to merge) that these documents are not connected to the top-level structure in the sidebar.  I will do something about that after the merge.  I believe all of our documents should be connected to "the structure" even though we are pretty likely to revise that structure extensively as the work progresses.

**Comment by JamieDawson** [comment]
@joshuaauerbachwatson 
> There are broken image links in workbench-playground-1-intro.md. The images should be in the repo and the links should be relative.

Lately, I've just been uploading my images to Flickr and then getting the links from there.  Do you believe we should create an images folder and collectively start adding the images there?

**Comment by rabbah** [comment]
We should put the images into our repo - is there a reason to use flickr? If not our repo then i would use a nimbella bucket instead, which is also something we control.

**Comment by joshuaauerbachwatson** [comment]
> Lately, I've just been uploading my images to Flickr and then getting the links from there. Do you believe we should create an images folder and collectively start adding the images there?

There would be no need to create another images folder.   There already is one in `static/img` in this repository.   When @niks3089 moved things over from `newdoc` and established the new structure, he put the images from some of my documents there, alongside some docusaurus ones, so I just assumed we'd continue that convention.

In fact, I am open to other possibilities but the links you have there now don't work (at least they didn't for me).   And I think it _is_ important to be consistent.  So, I believe you should put your images in `static/img` for now and we can separately discuss alternatives as things evolve.

**Comment by JamieDawson** [comment]
> There are broken image links in workbench-playground-1-intro.md. The images should be in the repo and the links should be relative.

Images added to `static/img`. Thank you for addressing that. I will no longer us Flickr.

> The document workbench-playground-3-deploy-from-github.md is misnamed. It is about a workbench feature but not a playground feature.

Renamed file.

**Comment by rabbah** [comment]
@JamieDawson please link these docs to the side bar.

**Comment by JamieDawson** [comment]
1) Documents are linked in the sidebar.
2) A `git rebase` and `git diff --origin workbench-documents` was applied to this repo for completion. 

Document ready for review.

File: docs/workbench-activations.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+```
+activation list
+```
+
+The `activation list` command will list the recent activations that have occurred. It will also display information about queueing delays, container initialization, execution time, and failures of activations.
+
+You can click on the individual IDs, which will automatically run `activation get` on them.
+
+![](https://user-images.githubusercontent.com/16840579/98278991-37434e00-1f4e-11eb-9ad1-3d8356ca37b6.gif)
```
Comment (rabbah) [inline docs/workbench-activations.md:20]: @JamieDawson this link doesn't work. You need to add the git as a file and make it part of this commit.
Then you can reference it as `./images/<filename>.gif` where filename is the name of the image. Please use a descriptive name.

File: docs/workbench-playground-1-intro.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+![](https://github.com/JamieDawson/docs/blob/workbench-documents/static/img/playground_languages.png)
+
+### Test your code by passing in arguments.
+
+- How to start testing:
+
+  - Click the run button to run the command and test the output.
+  - Change the value in `input parameters` to test the argument (For example, try changing “nimbella” to your name and see what happens)
+
+    ![](https://github.com/JamieDawson/docs/blob/workbench-documents/static/img/run_commands_in_playground.gif)
```
Comment (rabbah) [inline docs/workbench-playground-1-intro.md:29]: same re link.

File: docs/workbench-playground-1-intro.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+  - Click the publish button to get a secure domain for the function. Paste that domain into your browser. Test the arguements by adding `?name=nimbella` to the end of the URL.
+    ![](https://github.com/JamieDawson/docs/blob/workbench-documents/static/img/playground-test-publish-button-gif.gif)
```
Comment (rabbah) [inline docs/workbench-playground-1-intro.md:32]: same re link. Also notice `-gif.gif` redundancy.

File: docs/workbench-playground-1-intro.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+To display display the menu in the Workbench, run `help`.
+To remove the menu, run `clear`.
+
+## What you can do in the Playground?
+
+### Select the right programming language for you.
+
+At the top of the workbench you can click the button that says "JavaScript" and select from a wide range of programming languages. You can pick JavaScript, Typescript, Python, Swift, Java, Go, or PHP.
+
+![](https://github.com/JamieDawson/docs/blob/workbench-documents/static/img/playground_languages.png)
```
Comment (rabbah) [inline docs/workbench-playground-1-intro.md:20]: As noted above, add `playground_languages.png` as a file to this PR and commit it, then reference it using a relative link.

**Review by rabbah — CHANGES_REQUESTED** [review]

**Comment by JamieDawson** [comment]
All image fixes Rodric requested have been applied.
Pending review.

File: docs/workbench-deploy-from-GitHub.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
+---
+id: workbench-deploy-from-github
+title: Workbench
+sidebar_label: Workbench - Deploy from GitHub
+---
+
+# Deploy from GitHub.
+
+What's nice about the Workbench is that you can deploy your projects that are stored on your GitHub account.
```
Comment (rabbah) [inline docs/workbench-deploy-from-GitHub.md:9]:
```suggestion
What's nice about the Workbench is that you can deploy your projects directly from your GitHub repositories.
```

File: docs/workbench-deploy-from-GitHub.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+To run an example of a working `project deploy` command, run ``project deploy github:nimbella/demo-projects/visits` to deploy our page visitor demo app.
+
+**Note**: If you get an error saying you don't have permission to deploy from GitHub, run this command: `nim auth github --initial`. This command will only work if you have a GitHub account, which you will need to acquire separately.
+
+### Limitations for deploying from GitHub.
+
+Not all projects can be deployed from the Workbench.
+
+- Limitations:
+  "If the project requires building (see [ref](https://github.com/nimbella-corp/nimbella-cli/blob/master/newdoc/building.md)), the build will be run for you in the cloud. This may fail if the build exceeds the time limits or memory limits allotted by our remote build capability."
```
Comment (rabbah) [inline docs/workbench-deploy-from-GitHub.md:26]: The link here in `[ref]` is still wrong.

**Comment by JamieDawson** [comment]
Hey @rabbah , I believe I made a mistake with this.
After fixing the error "GitHub this branch has conflicts that must be resolved" which was above this post. I clicked the "Squash and merge" button not realizing I could do that on my own. I was under the impression that it would need a 2nd person to ok the merge request.

**Comment by rabbah** [comment]
It's OK. If there are remaining issues, you can handle them in another PR.
(We haven't enabled checks yet for convenience and you have write privileges and why you can merge your own PRs at this time.)

**Pull request merged**